packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -100,6 +114,13 @@ export interface ServerDeploymentGroupProps {
      * @default an auto-generated name will be used
      */
     deploymentGroupName?: string;
+
+    /**
+     * The EC2/on-premise Deployment Configuration to use for this Deployment Group.
+     *
```

Comment:
Since this is an interface. Provide some pointers/example on how to use a predefined one and how to define a custom one

Reply:
Good point. Will fix.